client/src/rootReducer.js

```diff
@@ -3,10 +3,21 @@ import { reducer as form } from 'redux-form';
 import application from './app/appReducer';
 import auth from './auth/authReducer';
 
-export const rootReducer = combineReducers({
+const appReducer = combineReducers({
   application,
   auth,
   form,
 });
 
+export const rootReducer = (state, action) => {
+  let newState = state;
+
+  if (action.type === 'USER_LOGOUT') {
```

Comment:
Why is this in the top level reducer? Does that let us nuke all state from all reducers by doing it here? 

In the non-prototype we'd be storing the session info in local storage so you could close the window and reopen it and still be logged in. We can ignore that here. 

At the least, we should be importing USER_LOGOUT from the consts file.